src/PrestaShopBundle/Resources/views/Admin/Sell/CustomerService/MerchandiseReturn/Blocks/product.html.twig

```diff
@@ -0,0 +1,78 @@
+{#**
+ * 2007-2020 PrestaShop SA and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2020 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ *#}
+
+{% set rowIsDisplayed =  (productIndex is defined and paginationNum is defined and productIndex > paginationNum) %}
+
+<tr id="merchandiseReturnProduct_{{ product.orderDetailId }}" class="cellProduct{% if rowIsDisplayed %} d-none d-print-table-row{% endif %}">
+  <td class="cellProductName">
```

Comment:
I think @NeOMakinG will require us to use kebab-case and meaningful names lile `class="merchandise-return-product-name"`

We must try to put class and IDs to allow easy selectors for jQuery

Reply:
yup, in fact we have a "convention" to use `kebab-case` for css classes and ids where possible (im not sure its written anywhere, but im sure it was decided somewhere in orders migration).

Reply:
> Found very small issues, the quality of this PR is very good !
> 
> The main worry I have is the pagination logic. I feel like we're doing something wrong here 
> 
> @zuk3975 am I being paranoid ? wdyt ?
The page currently looks like this: 
![Screenshot from 2020-05-28 17-49-19](https://user-images.githubusercontent.com/31609858/83157311-1f98ab80-a10c-11ea-9506-d8d5427a761e.png)

what i don't like is that there is a "save" button at the bottom, which implies that you will save the changes only after clicking it, but in reality if you delete something from list it is deleted instantly, so it doesn't look good in UX perspective.

What I hope could be done:
The page could be separated into two different parts (because these aren't dependent on each other anyway).
* the form (which really contains only status input)
* the grid of products.

So if we just separate these blocks, we can implement simple grid using our grid component (with all the hooks, and convenient simple php code) and a separate identifiable form for merchandise returns.

Unless I am missing something it sounds like a perfect decision :man_shrugging:

Reply:
> > Found very small issues, the quality of this PR is very good !
> > The main worry I have is the pagination logic. I feel like we're doing something wrong here
> > @zuk3975 am I being paranoid ? wdyt ?
> > The page currently looks like this:
> > ![Screenshot from 2020-05-28 17-49-19](https://user-images.githubusercontent.com/31609858/83157311-1f98ab80-a10c-11ea-9506-d8d5427a761e.png)
> 
> what i don't like is that there is a "save" button at the bottom, which implies that you will save the changes only after clicking it, but in reality if you delete something from list it is deleted instantly, so it doesn't look good in UX perspective.
> 
> What I hope could be done:
> The page could be separated into two different parts (because these aren't dependent on each other anyway).
> 
> * the form (which really contains only status input)
> * the grid of products.
> 
> So if we just separate these blocks, we can implement simple grid using our grid component (with all the hooks, and convenient simple php code) and a separate identifiable form for merchandise returns.
> 
> Unless I am missing something it sounds like a perfect decision 🤷‍♂️

Yes ! Great !

Indeed on Order page, we could not do this because the product rows were used as input for other forms. But here product rows are just products rows, right ?